demos/demo.cpp

```diff
@@ -147,16 +147,7 @@ int main()
   std::shared_ptr<Driver> driver_ptr = std::make_shared<Driver>();
   std::shared_ptr<BaseData> data_ptr = std::make_shared<SingleProcessData>();
 
-  // max time allowed for the robot to apply an action.
```

Comment:
@mwuethri I am actually a bit unsure now.  In this demo no real robot is involved, so technically no monitoring is needed.  However, maybe it makes sense to keep it here for demonstration purposes?